record.c

```diff
@@ -300,6 +303,16 @@ int janus_recorder_save_frame(janus_recorder *recorder, char *buffer, uint lengt
 		/* If media will be end-to-end encrypted, mark it in the recording header */
 		if(recorder->encrypted)
 			json_object_set_new(info, "e", json_true());
+		/* Write extmaps */
+		if(recorder->extmaps) {
+			json_t *ext_json = json_array();
+			GList *ext = recorder->extmaps;
+			while(ext != NULL) {
+				json_array_append_new(ext_json, json_string(ext->data));
+				ext = ext->next;
+			}
+			json_object_set(info, "x", ext_json);
```

Comment:
If the purpose is having mappings, here, this should be a key/value object, not a serialized string.